packages/block-editor/src/components/media-upload-progress/index.native.js

```diff
@@ -115,19 +117,26 @@ export class MediaUploadProgress extends Component {
 		}
 	}
 
+	getRetryMessage() {
+		if ( this.props.isConnected === true ) {
+			// eslint-disable-next-line @wordpress/i18n-no-collapsible-whitespace
+			return __( 'Failed to insert media.\nTap for more info.' );
+		}
+		return __( 'You are currently offline.' );
```

Comment:
We may want to expand this message to include another line like "Media uploads will resume when network connectivity has been restored." Leaving it as "You are currently offline." until that feature is implemented. Open to other suggestions, too.